serverlessworkflow/modules/ROOT/pages/cloud/operator/configuring-workflows.adoc

```diff
@@ -63,7 +63,7 @@ A few properties can not be changed in this configuration. Usually, they are alr
 |0.0.0.0
 |all
 
-|org.kie.kogito.addons.knative.health-enabled
+|org.kie.kogito.addons.knative.eventing.health-enabled
```

Comment:
Maybe add this change to the release notes already as this is a breaking change.

Reply:
It is not a breaking change. The documentation was actually incorrect.